roles/variables/defaults/main.yml

```diff
@@ -433,10 +433,6 @@ kafka_broker:
   appender_log_path: "{{kafka_broker_default_log_dir}}"
   # Deprecated way of providing custom properties
   properties: {}
```

Comment:
Can this also be deprecated since we are using kafka_broker_custom_properties now?

Reply:
We should. Probably a different PR would be better.